### ISSUE TITLE
[bug] set show option false fail

### ISSUE THREAD

**Issue by hurryhuang1007** [Seo-Rii/electron-acrylic-window#18]
hello~ this repo is awesome. But I encountered a problem.

The created window is displayed by default, even if `show: false` of option.
And if you call the hide method immediately after creating the window, the window will lost vibrancy effect.

This is how I solve this problem now
```js
const win = new BrowserWindow({
  show: false,
  movable: false,
  width,
  height,
  minWidth: 0,
  minHeight: 0,
  resizable: false,
  titleBarStyle: 'default',
  skipTaskbar: true,
  thickFrame: false,
  transparent: true,
  backgroundColor: '#00ffffff',
  vibrancy: vibrancy || (global.__IS_MAC__ ? 'sidebar' : 'appearance-based'),
  x: -800,
  y: -800,
});

if (global.__IS_WIN__) {
  win.once('ready-to-show', () => {
    setTimeout(() => win.hide(), 200);
  });
}
```
And my electron version is 8.2.5

**Comment by Seo-Rii** [comment]
Thank you for reporting Bug! I will fix the bug soon.

**Comment by Seo-Rii** [comment]
Resolved on a30c9ad(v0.3.3)!
